src/ansi.rs

```diff
@@ -1574,6 +1577,14 @@ where
             ('E', []) => handler.move_down_and_cr(next_param_or(1) as usize),
             ('F', []) => handler.move_up_and_cr(next_param_or(1) as usize),
             ('G', []) | ('`', []) => handler.goto_col(next_param_or(1) as usize - 1),
+            ('W', [b'?']) => {
+                if next_param_or(0) == 5 {
+                    const DEFAULT_INTERVAL: u16 = 8;
+                    handler.reset_tabs(DEFAULT_INTERVAL);
```

Comment:
The constant here doesn't really help and would never be changed. The escape itself specifies it as 8, so a terminal emulator with tabstops at every 16 columns should still set it to 8.

Reply:
Removed the constant and replaced it with a number